app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -209,5 +213,16 @@ public static String getIdFromStormpathHref(String href) {
         }
         return null;
     }
+    
+    public static DateTime getDateTimeOrDefault(String value, DateTime defaultValue) {
+        if (isBlank(value)) {
+            return defaultValue;
+        }
+        try {
+            return DateTime.parse(value);
+        } catch(Exception e) {
+            throw new BadRequestException(value + " is not an ISO timestamp");
+        }
+    }
```

Comment:
Was original in StudyController where it was tested, moved to DateUtils and added separate tests there.